src/Components/ADT3DSceneBuilder/Internal/Shared/MakePivotRequired.tsx

```diff
@@ -0,0 +1,26 @@
+import { IPivotItemProps } from '@fluentui/react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { customPivotItemStyles } from '../Behaviors/BehaviorsForm.styles';
+
+export function setPivotToRequired(
+    isValid: boolean | undefined,
+    link?: IPivotItemProps,
+    defaultRenderer?: (link?: IPivotItemProps) => JSX.Element | null
+): JSX.Element | null {
+    const { t } = useTranslation();
+    if (!link || !defaultRenderer) {
+        return null;
+    }
+    return (
+        <span className={customPivotItemStyles.root}>
```

Comment:
customPivotItemStyles should be part of this file, not imported from BehaviorsForm